changelog/updates/2024-09-12-etcd.md

```diff
@@ -0,0 +1 @@
+- etcd ([3.5.16](https://github.com/etcd-io/etcd/releases/tag/v3.5.16))
```

Comment:
Etcd has some wacky changelog situation - the tagged commit has no changelog, the release branches have no changelog. The changelogs for all the release branches are in the main branch, so let's point there instead.

```suggestion
- etcd ([3.5.16](https://github.com/etcd-io/etcd/blob/96e17d38e244ef18756a55d33066f2893402794b/CHANGELOG/CHANGELOG-3.5.md#v3516-2024-09-10))
```